contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+  <ComparisonRow column1={true} column2={true} feature="Migration" description="Migrate data from Google Analytics" />
+  <ComparisonRow column1={"50.6 kb"} column2={"~1 kB"} feature="Script size" description="Size of the script that runs on your site" />
+</ComparisonTable>
+
+## Product analytics
+
+Teams rely on [product analytics](/product-analytics) when they need more customization in both capturing and analyzing date. Although not a focus of Fathom, it does have offer some basic product analytics features with event tracking.
+
+<ComparisonTable column1="PostHog" column2="Fathom">
+  <ComparisonRow column1={true} column2={true} feature="Custom events" description="Capture and analyze any type of event" />
+  <ComparisonRow column1={true} column2={true} feature="Custom properties" description="Add more data to custom events or users" />
+  <ComparisonRow column1={true} column2={false} feature="Autocapture" description="Capture frontend clicks and events automatically" />
+  <ComparisonRow column1={true} column2={false} feature="Custom graphs and trends" description="Build custom insights and visualizations" />
+  <ComparisonRow column1={true} column2={false} feature="Funnels" description="Track users through a sequence of events" />
+  <ComparisonRow column1={true} column2={false} feature="Retention" description="Visualize which users stay, for how long" />
+  <ComparisonRow column1={true} column2={false} feature="Custom dashboards" description="Build a custom dashboard of your own or use a template" />
+  <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
+  <ComparisonRow column1={true} column2={false} feature="User profiles" description="View user demographic data such as country, language, and device type" />
+  <ComparisonRow column1={true} column2={false} feature="Group analytics" description="Track metrics at a company and account level" />
+  <ComparisonRow column1={true} column2={false} feature="Paths" description="Track user flows and where they drop off" />
+  <ComparisonRow column1={true} column2={false} feature="SQL" description="Query your data with SQL" />
+</ComparisonTable>
+
+## Platform
+
+Beyond web analytics is where PostHog really shines. We provide all of the tools developers need to build a successful product (and they all work together).
+
+<ComparisonTable column1="PostHog" column2="Fathom">
+  <ComparisonRow column1={true} column2={true} feature="EU hosting" description="Access and store your data in the EU" />
+  <ComparisonRow column1={true} column2={true} feature="API" description="Capture events, get stats, and make changes via API" />
+  <ComparisonRow column1="Invite teammates to collaborate for all features" column2="Share web analytics with or without a password" feature="Collaboration" description="Share your projects and sites with teammates" />
```

Comment:
Lets make sure this one looks good in the preview

Reply:
Seems like it looks alright. But I'm open to changes if there's some better way to communicate this.

Fathom basically allows dashboards to be private (default), public, or password protected ([source](https://usefathom.com/docs/features/shared-dashboards)).

<img width="788" alt="preview" src="https://github.com/user-attachments/assets/e2cfbca0-ed8d-4d70-a50c-fc2cf15bc832">